CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [3.2.0] - 2021-11-17
+
+### Changed
+
+- Updated HMRC banner to use SVG logo and tweaked vertical alignment
```

Comment:
@tomshaw1 / @timsb with this I can't see the ticket I think so it might have been explained already there, but are we sure the vertical alignment needed tweaking? I ask this as devils advocate just because I was looking at the "swipe" view of the github rich diff and I couldn't see much obvious misalignment previously 🤷‍♂️ just me curious, not a blocker 

<img width="302" alt="CleanShot 2021-11-17 at 16 04 45@2x" src="https://user-images.githubusercontent.com/100650/142236929-a1a4bf81-6e24-48ed-9959-5769dbfc4b09.png">

Reply:
The reason we changed it is we saw the difference between ours and the current banner, and noticed that the current banner does not have equal spacing between the top of the border and the top of the image, and bottom of the border and bottom of the image. Hope that makes sense

Reply:
yup that makes sense!